mods/screwdriver/init.lua

```diff
@@ -19,6 +11,73 @@ screwdriver.rotate_simple = function(pos, node, user, mode, new_param2)
 	end
 end
 
+-- For attached wallmounted nodes: returns true if rotation is valid
+-- simplified version of minetest:builtin/game/falling.lua#L148.
+local function check_attached_node(pos, rotation)
+	local d = minetest.wallmounted_to_dir(rotation)
+	local p2 = vector.add(pos, d)
+	local n = minetest.get_node(p2).name
+	local def2 = minetest.registered_nodes[n]
+	if def2 and not def2.walkable then
```

Comment:
No, it is not possible. See [there](https://github.com/minetest/minetest/blob/master/builtin/game/falling.lua#L164).